kaldb/src/main/java/com/slack/kaldb/metadata/dataset/DatasetMetadataStore.java

```diff
@@ -1,22 +1,22 @@
-package com.slack.kaldb.metadata.service;
+package com.slack.kaldb.metadata.dataset;
 
 import com.slack.kaldb.metadata.core.PersistentMutableMetadataStore;
 import com.slack.kaldb.metadata.zookeeper.MetadataStore;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ServiceMetadataStore extends PersistentMutableMetadataStore<ServiceMetadata> {
+public class DatasetMetadataStore extends PersistentMutableMetadataStore<DatasetMetadata> {
   public static final String SERVICE_METADATA_STORE_ZK_PATH = "/service";
```

Comment:
We should add a todo here explaining why this is still service, and what we expect to happen